udata_gouvfr/theme/templates/home.html

```diff
@@ -98,7 +98,7 @@ <h4 class="title">
 
             <div class="body col-sm-8">
                 <label>{{ _('News in data') }}</label>
-                <h3 class="title ellipsis-dot">
```

Comment:
No, it was working without the ellipsis. In fact it works better without:
- very long title can spread on 2 lines (it's better than the ellipsis on one, because what matter here is the news title)
- there is already an hidden overflow so the summary is already properly hidding